Presentation.Web/app/components/it-advice/it-advice-modal-dialog.ts

```diff
@@ -181,7 +206,7 @@
                         $scope.errMessage = "";
                         $scope.stopDateErrMessage = "";
                         $scope.curDate = new Date();
-                        if (!moment($scope.stopDate, "dd-MM-yyyy").isValid() ||
+                        if (!moment($scope.stopDate, allowedDateFormats).isValid() ||
```

Comment:
Vi skal lige have en den til at validere på "strict" format ellers tillader den for meget:
moment($scope.stopDate, allowedDateFormats, true)